python/GangaDirac/Lib/Utilities/DiracUtilities.py

```diff
@@ -213,6 +212,14 @@ def _checkProxy( delay=60, renew = True, shouldRaise = True, force = False ):
             _dirac_check_proxy( renew, shouldRaise )
             last_modified_time = time.time()
 
+class GangaDiracException(GangaException):
```

Comment:
Most exceptions types are named `FooError` rather than `FooException` both here in Ganga and in Python in general. Therefore I think naming this `GangaDiracError` would be better.